From cbef954dda4dbaa5f13c291041932cb7cf7da829 Mon Sep 17 00:00:00 2001
From: Will Dietz <will.dietz@sifive.com>
Date: Thu, 20 Oct 2022 10:16:23 -0500
Subject: [PATCH] Fixup pkgconfig file for absolute install dirs.

Fixes:
```
slang> Broken paths found in a .pc file! /nix/store/23f6pl4nhwl4xx1h35hs64d1kqh9g5sk-slang-1.0g20221020_fdf27a0/share/pkgconfig/sv-lang.pc
slang> The following lines have issues (specifically '//' in paths).
slang> 2:includedir="${prefix}//nix/store/23f6pl4nhwl4xx1h35hs64d1kqh9g5sk-slang-1.0g20221020_fdf27a0/include"
slang> 3:libdir="${prefix}//nix/store/23f6pl4nhwl4xx1h35hs64d1kqh9g5sk-slang-1.0g20221020_fdf27a0/lib"
slang> It is very likely that paths are being joined improperly.
slang> ex: "${prefix}/@CMAKE_INSTALL_LIBDIR@" should be "@CMAKE_INSTALL_FULL_LIBDIR@"
slang> Please see https://github.com/NixOS/nixpkgs/issues/144170 for more details.
error: builder for '/nix/store/4w5nq781aw161b9lrk1f59j914ibk368-slang-1.0g20221020_fdf27a0.drv' failed with exit code 1;
```
---
 scripts/sv-lang.pc.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/sv-lang.pc.in b/scripts/sv-lang.pc.in
index eabfbc9a..fdfacce8 100644
--- a/scripts/sv-lang.pc.in
+++ b/scripts/sv-lang.pc.in
@@ -1,6 +1,6 @@
 prefix=@CMAKE_INSTALL_PREFIX@
-includedir="${prefix}/@CMAKE_INSTALL_INCLUDEDIR@"
-libdir="${prefix}/@CMAKE_INSTALL_LIBDIR@"
+includedir="@CMAKE_INSTALL_FULL_INCLUDEDIR@"
+libdir="@CMAKE_INSTALL_FULL_LIBDIR@"
 
 Name: @PROJECT_NAME@
 Description: @PROJECT_DESCRIPTION@
-- 
2.37.3

